tests/OSPSuite.Infrastructure.Tests/Import/DataSourceSpecs.cs

```diff
@@ -164,6 +164,11 @@ protected override void Context()
          sut = new DataSource(_fakedImporter);
          sut.DataSets.Add("sheet1", _fakeDataSet);
       }
+
+      protected string getFirstMeasurementUnit()
+      {
+         return sut.DataSets.KeyValues.First().Value.Data.First().Data.ToList()[1].Value[1].Unit;
```

Comment:
This structure lol. Value.Data.Data.Value...

Reply:
@msevestre Yes I know - and totally agree. The only reason I would leave this here is that it is only the test and in the actual code it is never called like taht - the unit is deduced further down in the processesing.